doc/rst/language/archivedSpecs.rst

```diff
@@ -5,6 +5,7 @@ Documentation Archives
 
 Online Documentation Archives
 -----------------------------
+* `Chapel 2.2  <https://chapel-lang.org/docs/2.2/index.html>`_
```

Comment:
@arezaii : Noting for your reference (for next time) that to make our URLs more canonical, I'm removing the `index.html` part of the URLs in this list in an upcoming PR.